packages/raystack/v1/components/textArea/textArea.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { HTMLAttributes, PropsWithChildren } from "react";
+import { VariantProps, cva, cx } from "class-variance-authority";
+import { InfoCircledIcon } from "@radix-ui/react-icons";
+
+import styles from "./textarea.module.css";
+
+const textArea = cva(styles.textarea, {
+  variants: {
+    error: {
+      true: styles.error
+    }
+  }
+});
+
+export interface TextAreaProps extends PropsWithChildren<VariantProps<typeof textArea>>, 
+  HTMLAttributes<HTMLTextAreaElement> {
+  label?: string;
+  isOptional?: boolean;
```

Comment:
why cant we use `required` props here?
we are using `required` in other components

Reply:
This was used in the figma design. I agree that required makes more sense and will be consistent with other component. Changing to required.